api/v1beta1/types.go

```diff
@@ -243,6 +243,9 @@ type LoadBalancerSpec struct {
 	// FrontendIPsCount specifies the number of frontend IP addresses for the load balancer.
 	// +optional
 	FrontendIPsCount *int32 `json:"frontendIPsCount,omitempty"`
+	// BackendPool specifies the name of backend pool for the load balancer.
+	// +optional
+	BackendPool *string `json:"backendPool,omitempty"`
```

Comment:
Might we ever want to add more configuration for the backend pool? Perhaps this should be a struct with a single member for now so that we can extend it in the future with other configuration

```
backendPool:
  name: ...
```